index.html

```diff
@@ -21,16 +21,12 @@ <h1 class="post-title">
 
 <div class="pagination">
   {% if paginator.next_page %}
-    <a class="pagination-item older" href="{{ site.baseurl }}page{{paginator.next_page}}">Older</a>
+    <a class="pagination-item older" href="{{paginator.next_page_path | prepend: site.baseurl | replace: '//', '/' }}">Older</a>
```

Comment:
I took the `replace` code straight from the [jekyll documentation for pagination](http://jekyllrb.com/docs/pagination/) (bottom of the page).  But I'm fine with removing it.